install/kubernetes/helm/istio/values-envoyv2-transition.yaml

```diff
@@ -3,7 +3,8 @@ global:
   nodePort: true
   proxy:
     image: proxyv2
-
```

Comment:
This is used in the NO AUTH test - only set it in the auth test ( where both mtls and control plane tls are enabled)